src/app/app.module.ts

```diff
@@ -12,6 +12,7 @@ GNU General Public License for more details.
 You should have received a copy of the GNU General Public License
 along with OVFPlayer.  If not, see <https://www.gnu.org/licenses/>.
 ::END::LICENCE:: */
+import 'hammerjs';
```

Comment:
What's this used for?

Reply:
This is for gesture support.  It could be very useful for other gesture stuff....
The material slider needs it to actually slide, rather than just moving to where you click